python/ray/train/checkpoint.py

```diff
@@ -274,6 +274,18 @@ def best_checkpoint_path(self) -> Optional[Path]:
         else:
             return None
 
+    @property
+    def latest_checkpoint_id(self) -> Optional[int]:
```

Comment:
This is a property of `CheckpointManager` which isn't exposed in the docs! We want it to be public to `CheckpointManager` so that `Trainer` can access it.